README.md

```diff
@@ -1,12 +1,76 @@
-# Webpack Boilerplate
+# Pokemon
 
->  A simple yet powerful webpack boilerplate, which you can later use as a starting point in your projects.
+>  **Pokemon** is a webapp that simulates a pokedex showing the pokemon of a certain type. The user can choose the type of pokemon they want to see.
+> The app will show the pokemon of that type. The use can see the name, type, and image of the pokemon. The user can also see the games the pokemon is in.
 
 
-## Features
 
-- Webpack setup 
-- HtmlWebpackPlugin
-- Loading CSS setup
-- Webpack-dev-server
 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+## Built With
+
+- HTML, CSS, and Javascripts
+- Node.js
+- Web-hint, Stylelint
+- Boostrap
+- Jest
+- Webpack
+
+### Click link below for a live demo
+[Demo Link](https://github.com/Fourteen98/Pokemon/dist/)
+## Getting Started
+
+### Prerequisites
+install git, nodejs, any code editor in your local machine
+
+### Setup
+clone the repo or download from https://github.com/Fourteen98/Pokemone.git
+
+### Install dependencies
+
+`npm install`
+
+
+### Build the app
+
+`npm run build`
+
+
+### Run the app
+
+`npm start `
+This automatically start app on localhost:3000
+
+### Deployment
+
+To run the project run the "index.html" file in your browser.
+
+👤 **Author 1**
+
+- GitHub: [@Fourteen98](https://github.com/Fourteen98)
+
+👤 **Author 2**
+
+- GitHub: [@aizjicod](https://github.com/aizjicod)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Great job adding the `MIT.md` file 💯, but you forgot to customize it, so kindly add you name and customize it 👌

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯


![image](https://user-images.githubusercontent.com/45231276/181859787-239d3368-3836-4163-94b9-97cbf3985fbd.png)